seed/get-drupal-services.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch');
+const fs = require('fs');
+const resolve = require('path').resolve;
+
+const collectionsPath = resolve(__dirname, 'data', 'services');
+
+const drupalUrl = process.env.DRUPAL_URL || 'http://inventive-d8-txc.pantheonsite.io';
+const url = `${drupalUrl}/rest/v1/fulfillments/services`;
```

Comment:
These are not the service categories. these are the services themselves. We need to get the service categories that I link to in the story, not the services. These service categories are used to identify the types of services that a service provides.

![image](https://user-images.githubusercontent.com/2193655/58955005-5d186200-8760-11e9-9d7b-1912186179c3.png)